messaging/src/test/java/org/axonframework/eventhandling/pooled/WorkPackageTest.java

```diff
@@ -73,6 +73,7 @@ class WorkPackageTest {
     private List<TrackerStatus> trackerStatusUpdates;
     private Predicate<TrackedEventMessage<?>> eventFilterPredicate;
     private Predicate<List<? extends EventMessage<?>>> batchProcessorPredicate;
+    private boolean cacheCleared;
```

Comment:
If we rename the operation on the `EventHandlerInvoker`, I anticipate we rename the field name too.
Perhaps something like `shutdownActionInvoked` (as present in the `Coordinator`) or `abortActionInvoked` would make sense.

Reply:
It's now done from the `Coordinator` on releasing the token. So no changes to the `Workpackage` are needed anymore.